lib/include/ultrahdr/icc.h

```diff
@@ -238,7 +244,7 @@ struct ICCHeader {
 class IccHelper {
  private:
   static constexpr uint32_t kTrcTableSize = 65;
-  static constexpr uint32_t kGridSize = 17;
+  static constexpr uint32_t kGridSize = 11;
```

Comment:
I wonder where is this change coming from. The original value is aligned with skia: https://cs.android.com/android/platform/superproject/+/android14-qpr3-release:frameworks/native/libs/ultrahdr/include/ultrahdr/icc.h;l=218?q=kGridSize

Reply:
I have updated this basing on this,
https://cs.android.com/android/platform/superproject/main/+/main:external/skia/src/encode/SkICC.cpp;drc=1cce66c0004230c737a7ef3bbc1559015d83eaa6;l=719

currently the skICC.cpp and icc.cpp are different. I have taken this change, as it reduces the encoded size.